lightstep/sdk/metric/aggregator/aggregation/temporality.go

```diff
@@ -57,3 +61,13 @@ func (t Temporality) Valid() bool {
 	}
 	return false
 }
+
+func ParseTemporality(str string) (Temporality, bool) {
+	switch strings.ToLower(str) {
+	case "cumualtive":
```

Comment:
typo

s/cumualtive/cumulative/

Reply:
Is there a `const` that exists for these already?

Reply:
lol guess I need a test!

Reply:
Tested!